src/app/content/utils/archiveTreeUtils.ts

```diff
@@ -39,3 +39,36 @@ export const findArchiveTreeSection = (
     stripIdVersion(section.shortId) === stripIdVersion(pageId)
     || stripIdVersion(section.id) === stripIdVersion(pageId)
   );
+
+export const prevNextBookPage = (
+  book: {tree: ArchiveTree},
+  pageId: string
+): {} | undefined => {
+  const flattenTree = flattenArchiveTree(book.tree);
+  const currentSection = findArchiveTreeSection(book, pageId);
+
+  const sections: { [s: string]: object; } = {};
```

Comment:
a stronger type for this would be:
```
{
    prev?: LinkedArchiveTreeSection;
    next?: LinkedArchiveTreeSection;
}
```

same as in the component